app/main.py

```diff
@@ -1,3 +1,58 @@
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: int) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: "Distance") -> "Distance":
+        if isinstance(other, Distance):
+            return Distance(self.km + other.km)
+        else:
+            return Distance(self.km + other)
+
+    def __iadd__(self, other: "Distance") -> "Distance":
+        if isinstance(other, Distance):
+            self.km += other.km
+        else:
+            self.km += other
+        return self
+
+    def __mul__(self, other: int) -> "Distance":
+        return Distance(self.km * other)
+
+    def __truediv__(self, other: int) -> "Distance":
```

Comment:
The `__truediv__` method assumes `other` is always an `int`. Consider adding a type check and raising a `TypeError` if `other` is not an `int`. Also, handle division by zero to prevent runtime errors.